app/controllers/mock.php

```diff
@@ -646,6 +649,69 @@
         $response->noContent();
     });
 
+App::get('/v1/mock/github/callback')
+    ->desc('Create installation document using GitHub installation id')
+    ->groups(['mock', 'api', 'vcs'])
+    ->label('scope', 'public')
+    ->param('installation_id', '', new Text(256, 0), 'GitHub installation ID', true)
+    ->param('projectId', '', new Text(2048), 'Project ID of the project where app is to be installed', true)
+    ->inject('gitHub')
+    ->inject('project')
+    ->inject('response')
+    ->inject('dbForConsole')
+    ->action(function (string $providerInstallationId, string $projectId, GitHub $github, Document $project, Response $response, Database $dbForConsole) {
```

Comment:
`string $providerInstallationId`
It's best to have variable name same as parameter name, which is currently `installation_id`.

Let's call param `providerInstallationId` as well. After this change, you may need to update some tests to use new param name.